src/test/java/synapseawsconsolelogin/AuthTest.java

```diff
@@ -110,29 +110,54 @@ public void testGetSSMParameter() {
 		Assume.assumeNotNull(credentials, credentials.getAWSAccessKeyId(), credentials.getAWSSecretKey());
 		
 		String propertyName = UUID.randomUUID().toString();
+		String ssmKey = UUID.randomUUID().toString();
 		String propertyValue = UUID.randomUUID().toString();
 		
 		Auth auth = new Auth();
 		
 		// the property has NOT been stored yet
 		assertNull(auth.getProperty(propertyName, false));
 		
+		System.setProperty(propertyName, "ssm::"+ssmKey);
+		
 		// now let's store the property in SSM
 		try {
 			AWSSimpleSystemsManagement ssmClient = AWSSimpleSystemsManagementClientBuilder.defaultClient();
 			
 			PutParameterRequest putParameterRequest = new PutParameterRequest();
-			putParameterRequest.setName(propertyName);
+			putParameterRequest.setName(ssmKey);
 			putParameterRequest.setValue(propertyValue);
 			putParameterRequest.setType(ParameterType.SecureString);
 			ssmClient.putParameter(putParameterRequest);
 		} catch (AmazonClientException e) {
 			// cannot continue with this integration test
 			return;
 		}
+	}
```

Comment:
Should we not check that we can get the property back?

Reply:
Yes, that was an unintentional deletion.  I just restored in in a new commit to this PR.